tests/small/gamma_condition_and.bril

```diff
@@ -0,0 +1,40 @@
+@main() {
```

Comment:
Your test is getting constant folded because you didn't include any arguments to main. You should prevent this

Reply:
Would also be cool to have a benchmark that actually benefits from your splitting optimization

Reply:
I fixed the constant folding. I don't tihnk that the split if statement can be extracted with our current optimizations, but I left an optimized bril program in the file to search for in the egraph once we add those kinds of tests.